debugfs/dump.c

```diff
@@ -262,10 +262,15 @@ static void rdump_symlink(ext2_ino_t ino, struct ext2_inode *inode,
 	free(buf);
 }
 
+typedef struct rdump_dirent_private {
```

Comment:
An anonymous struct would do (it's already used elsewhere):

```
typedef struct {
...
} rdump_dirent_private;
```

and replace `rdump_dirent_private_t` with `rdump_dirent_private` below.